framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -1171,12 +1178,18 @@ public void addAnnotation(AnnotationMirror a) {
         }
 
         /**
-         * Sets the parameter types of this executable type, excluding the receiver.
+         * Sets the parameter types and updates varargsType of this executable type, excluding the
+         * receiver. If paramTypes has been computed and this type is a varargs method, set
+         * varargType before calling this method.
```

Comment:
Should be `{@link varargsType}`? Or actually, do you still update varargsType here?